### DISCUSSION TITLE
What does "101" mean?

### DISCUSSION THREAD

**Discussion by nektobit** [alpinejs/alpine#1740] [1. Help]
Can anyone please tell why Alpine.js is now called "101"? What is the magic of these numbers? I saw a vote for a new name somewhere (and it seems that I even took part in it), but I lost the url. Please, what does "101" mean?

**Comment by SimoTod** [comment]
https://en.m.wikipedia.org/wiki/101_(topic)

**Reply by nektobit** [reply]
Thank you very much, so this refers to the documentation and not to the project itself 🤦‍♂️. I didn't know about such a term before. :)

**Comment by calebporzio** [comment]
Sorry for using such a localized term. I changed it to "Start Here" instead: https://alpinejs.dev/start-here
